packages/property/src/Charcoal/Property/ImageProperty.php

```diff
@@ -10,6 +10,7 @@
 // From 'charccoal-translator'
 use Charcoal\Translator\Translation;
 // From 'charcoal-property'
+use Charcoal\Property\Event\PropertyEvent;
```

Comment:
The Event Disptacher being [added](https://github.com/charcoalphp/charcoal/commit/3ba892085a25eac453d01f701370fed1e04f9f55) and then [removed](https://github.com/charcoalphp/charcoal/commit/372678845162cd84408389bba0e414a6142b3cec), I don't think we need the PropertyEvent use.